doc/javascripts/main.js

```diff
@@ -117,21 +102,15 @@ function setupSidebar() {
       var item = document.createElement("li");
       var p = document.createElement("p");
```

Comment:
Can I suggest that rather than creating a `p` here, you append a text node to the inside of the `a` instead?

Then you could remove the `p` in the HTML (see my code review comment above) I believe.